samples/testing/dump_operation/src/SWAP.qs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/// # Sample
+/// Multi File Testing Project
+///
+/// # Description
+/// Organizing code into multiple Q# source files is an important part of
+/// writing readable and maintainable code. In this project, we have `BellState.qs`, `SWAP.qs`,
+/// and `OperationEquivalence.qs`, which contain the operation to be tested.
+/// The presence of a Q# manifest file (`qsharp.json`) tells the compiler
+/// to include all Q# files under `src/`.
+/// These will be tested by Python wrapper, `test_dump_operation.py`
+
+namespace SWAP {
```

Comment:
This is a bit of a nitpick, but it can be a little confusing when the namespace has the same name as an intrinsic gate. Maybe `Test_SWAP` could avoid any conceptual overlap with the operation `SWAP`?